usr/bs_longhorn.c

```diff
@@ -132,6 +135,56 @@ static void bs_longhorn_request(struct scsi_cmd *cmd)
 		lh_client_close_conn(old_conn);
 		lh_client_free_conn(old_conn);
 		break;
+	case UNMAP:
```

Comment:
Can you link to the iscsi RFC for UNMAP since it's hard of the top of the head to know what the iscsi protocol message for UNMAP looks like :)

For fellow reviewer here is some reference for unmap protocol, might be useful.
- https://github.com/rancher/tgt/blob/longhorn_oct2019/usr/scsi.c
- https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf

Reply:
Sure. Actually, I also referred this Seagate manual during implementation. I will add the link to the comment.